devbox.json

```diff
@@ -38,6 +38,8 @@
     "ginkgo":                               "2.22.2",
     "act":                                  "0.2.72",
     "cloudfoundry-cli":                     "8.8.1",
+    "redocly":                              "latest",
```

Comment:
This was added https://github.com/cloudfoundry/app-autoscaler-release/commit/38d9bf47a5e94a1eea64b2fdcd44112670c2696d

Reply:
you are right this is probably an issue when merging, I removed this redundancy here: 3944726a5adac68617af69e55d97dbd7fb7381f2